config/activity_labels/recipe_coffee.yaml

```diff
@@ -0,0 +1,101 @@
+version: "1"
+labels:
+  # Item:
+  # - id: Integer identification number of the label.
+  # - label: Short human-readable semantic string to succinctly represent the
+  #          label.
+  # - step_detail: Longer descriptive string detailing the labeled step. This
+  #                currently acts as the step "instruction," but it's presence
+  #                here is questionable.
+  - id: 0
+    label: "background"
+    step_detail: "Background"
+  - id: 1
+    label: "measure-12oz-water"
+    step_detail: "Measure 12 ounces of water in the liquid measuring cup"
+  - id: 2
+    label: "pour-water-kettle"
+    step_detail: >-
+      Pour the water from the liquid measuring cup into the electric
+      kettle
+  - id: 3
+    label: "turn-kettle-on"
+    step_detail: "Turn on the Kettle"
+  - id: 4
+    label: "place-dipper-on-mug"
+    step_detail: "Place the dripper on top of the mug"
+  - id: 5
+    label: "filter-fold-1"
+    step_detail: >-
+      Take the coffee filter and fold it in half to create a semi-circle
+  - id: 6
+    label: "filter-fold-2"
```

Comment:
filter-fold-quarter
fold-1 and fold-2 sort of conveys something about order as if "this is the folding that happens first" and then "this is the folding that happens second." The activities need to be identifiable given only the content of the short video segment.

Reply:
Sure. The previous choice was arbitrary so I'll go with your suggestion here.